src/ast/passes/link.cpp

```diff
@@ -0,0 +1,88 @@
+#include <cstring>
+#include <fstream>
+
+#include "ast/passes/codegen_llvm.h"
+#include "ast/passes/link.h"
+#include "bpfbytecode.h"
+#include "util/temp.h"
+
+namespace bpftrace::ast {
+
+char LinkError::ID;
+void LinkError::log(llvm::raw_ostream &OS) const
+{
+  OS << "linking " << origin_ << ": " << strerror(err_);
+}
+
+Pass CreateExternObjectPass()
```

Comment:
Why does this pass live here?

Reply:
It is a placeholder. This will be exported in the future by the pass that resolves imports.